### PR TITLE
Add Sematext Logging & Monitoring to the user guide

### PR DESCRIPTION
**Pull request by megastef**
Added instructions for Monitoring and Logging with Sematext in the section _Monitoring, Logging, and Debugging Containers_

Background info: 
http://blog.kubernetes.io/2016/11/kubernetes-containers-logging-monitoring-with-sematext.html
Kubernetes partner: https://kubernetes.io/partners/
"_Sematext Logging & Monitoring: Automatic collection and processing of Metrics, Events and Logs for auto-discovered pods and Kubernetes nodes._"

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2364)

### CODE REVIEW

**Comment by chenopis** [comment]
@pwittrock Would you mind assigning this to an appropriate eng? Thx

**Comment by pwittrock** [comment]
@devin-donnelly @jaredbhatti Should this be hosted on our site?  Where do docs for partner stuff go?  Who in the community will be maintaining this?

**Comment by otisg** [comment]
Could these docs just be in the general Kubernetes monitoring/logging docs?  The hope is that they will help any Kubernetes user looking for monitoring/logging info or solutions.  If this howto gets tucked away under some Partners page nobody who is actually looking for information about monitoring/logging will find it - nobody looks under Partners pages for how-tos :)

**Comment by chenopis** [comment]
@megastef BTW, the Travis CI check is failing because of some unrelated change that has been fixed. If you merge `master` into this branch again, that should clear the Travis CI checks.

**Comment by megastef** [comment]
Thx @chenopis

**Comment by chenopis** [comment]
@devin-donnelly @jaredbhatti Was there any resolution to this?

**Comment by megastef** [comment]
I can't resolve the conflict.

**Comment by chenopis** [comment]
@megastef Sorry, we have been reorganizing the content and spitting the User Guides into Concepts, Tasks, and Tutorials. Consequently, I recommend putting sematext-logging-monitoring.md into `/docs/tasks/debug-application-cluster/`. So you will need to add it to the ToC in `_data/tasks.yml` instead.

**Comment by chenopis** [comment]
@megastef If you can check the Allow Edits by Maintainers check box, that will give me permission to make changes and resolve the conflict for you. Thx

**Comment by piosz** [comment]
@fgrzadkowski and I will review from the tech POV.

**Comment by megastef** [comment]
@chenopis I changed permission. Thx!

**Comment by chenopis** [comment]
@megastef I'll work on resolving the merge issue and will ping you when it's ready.

**Comment by chenopis** [comment]
@megastef Ok, this should be back on track. Carry on. =)

**Comment by chenopis** [comment]
@piosz @fgrzadkowski Is this PR still under review?

**Comment by piosz** [comment]
@megastef thanks for your PR. Apologies for such a huge delay. I'm happy that Sematext is integrated with Kubernetes!

We proposed [Monitoring Vision](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/monitoring_architecture.md) some time ago and we would like to revisit logging as well. The main change is that Kubernetes will not come with out of the box monitoring/logging solution (except from tiny system metrics monitoring) but rather make a room for 3rd party monitoring/logging solutions. This means that also there won't be any provider specific documentation in the Kubernetes docs but only generic architecture and links to 3rd party documentation. Due to historical reasons we still have out of the box support for Elasticsearch, InfluxDB and Stackdriver but the plan is to move them out of the Kubernetes org (there is even some progress https://github.com/kubernetes/kubernetes/pull/42127).

The bottom line is that instead of having a Sematext specific documentation in Kubernetes documentation, how about linking to your documentation instead (like https://sematext.com/container-support/#kubernetes-support)? If you agree we can find a good spot in the documentation for it.

BTW there is [SIG Instrumentation](https://github.com/kubernetes/community/tree/master/sig-instrumentation) which owns work related to monitoring and logging in Kubernetes, including integration with 3rd party solutions. Are you interested in giving a short demo of Sematext during one of the weekly meetings?

**Comment by steveperry-53** [comment]
@fgrzadkowski @otisg @piosz Is one of you ready to git this a Tech Review LGTM?

Second question:
Is there consensus on whether this belongs in the Kubernetes docs or on a third-party site? If this topic is not going to be in the Kubernetes docs, let's close this PR, and open a new PR for the purpose of linking to the content. Thanks.

**Comment by piosz** [comment]
Closing the PR

**Comment by otisg** [comment]
@piosz is there a new ticket for linking this content?  Or an issue for moving k8s docs elsewhere, so we can get docs linked?  Thanks!
